config/kubernetes/sig-docs/teams.yaml

```diff
@@ -85,14 +101,14 @@ teams:
     - ianychoi
     privacy: closed
     teams:
-      sig-docs-ko-reviews:
-        description: Reviews for Korean docs PRs
-        maintainers:
-        - ClaudiaJKang
-        - gochist
-        members:
-        - ianychoi
-        privacy: closed
+  sig-docs-ko-reviews:
```

Comment:
Thanks for fixing this, @rlenferink!

@cblecker @fejta We need `ko-reviewers` to _not_ be nested under `ko-owners` because child teams inherit parents' teams permissions iirc. Can you sanity check that changing the indentation will remove this as a child team and not have any side effects?

Reply:
I'm not sure about the no side effects part... Worst case it will delete the team ID and then recreate a new team and add the same people to it.